src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -301,7 +301,7 @@ UniversalDiscoveryModule.defaultProps = {
     loadContentTypes,
     findContentBySearchQuery,
     findLocationsByParentLocationId,
-    canSelectContent: () => true,
+    canSelectContent: (item, callback) => callback(false),
```

Comment:
Maybe: `true`? Previously we were always allowing to select a content.

Reply:
It is this case. `Callback` expects boolean if the button should be disabled. `false` means `do not disable button`.